elkm1_lib/keypads.py

```diff
@@ -20,7 +23,18 @@ def __init__(self, index: int, connection: Connection, notifier: Notifier) -> No
         self.last_user = -1
         self.code = ""
         self.last_keypress: Optional[tuple[str, int]] = None
+        # Not sure what these init values should be? should it be same as above?
+        self.chime_mode = None
+        self.last_function_key = None
 
+    def press_function_key(self,key: str) -> None:
+        """(Helper) Send a function key (1, ... 6, *, C)"""
+        self._connection.send(kf_encode(self.index,key))
+
+    # Should we create these for all function keys?
```

Comment:
No. Do you even need this function? Could it be left as `press_function_key(FunctionKeys.CHIME)`?